.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: python
 python:
   - "2.7"
-  - "3.3"
   - "3.4"
```

Comment:
I thought we weren't supporting 3.4 anymore

Reply:
Point well made didn't think this through, updated PR.